docs/tutorials/picorv32_ram.rst

```diff
@@ -0,0 +1,244 @@
+
+See the :ref:`Installation <Installation>` section for information on how to install SiliconCompiler, and the :ref:`Remote Processing <Remote Processing>` section for instructions on setting up the remote workflow.
+
+Download PicoRV32 Verilog Code
+------------------------------
+
+The heart of any digital design is its HDL code, typically written in a language such as Verilog or VHDL. High-level synthesis languages are gaining in popularity, but most of them still output their final design sources in a traditional HDL such as Verilog.
+
+PicoRV32 is an open-source implementation of a small RISC-V CPU core, the sort you might find in a low-power microcontroller. Its source code, license, and various tooling can be found `in its GitHub repository <https://github.com/YosysHQ/picorv32>`_.
+
+We have a copy of the core CPU module's Verilog at `examples/picorv32/picorv32.v <https://github.com/siliconcompiler/siliconcompiler/blob/main/examples/picorv32/picorv32.v>`_. Create a new directory for this project, and copy the ``picorv32.v`` file into it.
+
+Build the PicoRV32 Core using SiliconCompiler
+---------------------------------------------
+
+Before we add the complexity of a RAM macro block, let's build the core design using the open-source :ref:`Skywater 130 <skywater130_demo>` PDK. Copy the following build script into the same directory which you copied ``picorv32.v`` into:
+
+.. code-block:: python
+    :caption: <project_dir>/picorv32.py
+
+    import siliconcompiler
+
+    chip = siliconcompiler.Chip('picorv32')
+    chip.load_target('skywater130_demo')
+    chip.input('picorv32.v')
+    chip.set('option', 'remote', True)
+    chip.run()
+
+If you run that example as a Python script, it should take approximately 20 minutes to run if the servers are not too busy. We have not added a RAM macro yet, but this script will build the CPU core with I/O signals placed pseudo-randomly around the edges of the die area. Once the job finishes, you should receive a screenshot of your final design, and a report containing metrics related to the build in ``build/picorv32/job0/report.pdf``. SiliconCompiler will try to open the file after the job completes, but it may not be able to do so if you are running in a headless environment.
+
+.. image:: ../_images/picorv32_report.png
+
+For the full GDS-II results and intermediate build artifacts, you can install the EDA tools on your local system, and run the same Python build script with the :keypath:`option, remote` parameter set to ``False``.
+
+.. note::
+
+    We are not returning the full results during this early beta period because we want to minimize bandwidth, and we believe that our public beta is currently best suited for rapid prototyping and design exploration.
```

Comment:
@gadfort and @WRansohoff this sound like we will be providing more results in a remote run later. 
What additional results will be provided later?

Reply:
maybe make the note: note we are only returning a limited amount of information (and point them to the reprocessing for more details)